src/s3a_the_gutter.tw2

```diff
@@ -1,174 +1,905 @@
+
+(set: $gutterEndScenesViewed to 0)(set: $golfCartWreckViewed to false)(set: $annaLetterRead to false)
+
+::The Gutter - Ending hub
+I stood planning my next move in the crisp night air just outside of The Gutter's halo of dirty light.
+
+{(if: $gutterEndScenesViewed is 2)[[[I headed for the Church with all the grim determination I could muster|Church - Intro post-letter]].]}
+{(if: $golfCartWreckViewed is false)[>>[[I heard the noise of a familiar assclown behind me and turned to look|The Gutter - Ron's cart finale]].]}
+{(if: $annaLetterRead is false)[>>[[I ripped the envelope from my pocket and devoured Anna’s letter|The Gutter - Anna's letter]].]}
+
+::The Gutter - Ron's cart finale
+My eyes caught Ron's lumpy shape staggering through the darkness towards his discount battle wagon. This was too good to miss, so I posted up behind a tree with a good view and peeped out.
+
+Ron weaved down the hill to where he parked the cart. When he got there, he first flopped on the hood for a minute or two before [remembering how to get in]<cartStart|.
+
+(click: ?cartStart)[(if: $cartKeyTaken is true)[$dissolve[(display: "The Gutter - Ron's car finale - No key")]](else-if: $cartDamage is true)[$dissolve[(display: "The Gutter - Ron's car finale - Car damage")]](else:)[$dissolve[(display: "The Gutter - Ron's car finale - No damage")]]]
+
+(set: $gutterEndScenesViewed to it +1)(set: $golfCartWreckViewed to true)
+
+::The Gutter - Ron's car finale - Car damage
+I watched with glee as Ron sat down on the barbed wire piled in his seat and jumped up screaming. I almost blacked out laughing when he used his shirt as gloves to remove the wire.
+
+When Ron finally got his ass sat in the cart, it immediately sank further into the ground. He looked to the side and noticed the lack of wheels. “Fucking junkers!” he screamed as I began to cry from laughing.
+
+Next, Ron’s combination of anger and drunkenness led him to beat on the horn, which made desperate puffs of air like someone farting on a run to the bathroom. My tears of joy overflowed and soaked the earth.
+
+When Ron attempted to start the car, thick smoke poured out of the engine and into his face. He screamed for a while before passing out. After a few minutes without movement, I snuck down to the cart and wrote “Piss Baby” on his forehead.
+
+[[I headed off into the night|The Gutter - Ending hub]] towards a future that could only disappoint me in comparison.
+
+::The Gutter - Ron's car finale - No damage
+Ron revved the engine and then started madly honking on the horn. I watched him spin out, try to do a donut, turn the cart over, push the cart back up, start the cart again, and then speed off into the night at a blistering 12mph.
+
+As Ron’s cart crested a hill, it clipped a large rock. I saw the cart rise and flip beautifully like an angel underwater. If I could have one image to ruminate on for the rest of my life, I would choose this one.
```

Comment:
To double down on the beauty and grace of this moment:

>I watched the cart pirouette in beautiful slow-motion, like an angel underwater.

Reply:
Ayyy, that's a spicy edit! I'm terrible with awkward phrasing in my first drafts, and I fixed a bunch while coding this, but there's certainly still some swimming about. Thanks for improving this!